guides/common/modules/proc_configuring-the-project-client-in-keycloak.adoc

```diff
@@ -0,0 +1,44 @@
+[id="configuring-the-project-client-in-keycloak_{context}"]
+= Configuring the {Project} Client in {Keycloak}
+
+Use this procedure to configure the {Project} client in the {Keycloak} web UI and create group and audience mappers for the {Project} client.
+
+.Procedure
+
+. In the {Keycloak} web UI, navigate to *Clients* and click the {Project} client.
+. Configure access type:
+* If you want your users to authenticate to {Project} using the web UI, from the *Access Type* list, select *confidential*.
+* If you want your users to authenticate to {Project} using the CLI, from the *Access Type* list, select *public*.
+. In the *Valid redirect URI* fields, add a valid redirect URI.
++
+* If you want your users to authenticate to {Project} using the web UI, in the blank field below the existing URI, enter a URI in the form `https://{foreman-example-com}/users/extlogin`. Note that you must add the string `/users/extlogin` after the {Project} FQDN.
++
+After completing this step, the {Project} client for logging in using the web UI must have the following *Valid Redirect URIs*:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+https://{foreman-example-com}/users/extlogin/redirect_uri
+https://{foreman-example-com}/users/extlogin/
```

Comment:
this should be 'https://{foreman-example-com}/users/extlogin'

Reply:
This is a good catch!